components/email_aliases/email_aliases.mojom

```diff
@@ -18,6 +18,8 @@ struct AuthState {
   AuthenticationStatus status;
   // The authentication email address for this current account
   string email;
+  // Error message if authentication has failed somehow
+  string? error_message;
```

Comment:
should email be optional? I feel like we can only have either an error or an email, but maybe I'm wrong?

Reply:
It's still possible to have an `email` which is showing the current account email address we are attempting to authenticate with. It is displayed in the kAuthenticating mode.